src/Console/ConfigurationResolver.php

```diff
@@ -314,13 +314,15 @@ private function computeConfigFiles()
         }
 
         $path = $this->path;
-        if (is_file($path) && $dirName = pathinfo($path, PATHINFO_DIRNAME)) {
-            $configDir = $dirName;
-        } elseif ($this->isStdIn || null === $path) {
+
+        if ($this->isStdIn || 0 === count($path)) {
             $configDir = $this->cwd;
-            // path is directory
+        } elseif (1 < count($path)) {
+            throw new InvalidConfigurationException('For multiple paths config parameter is required.');
```

Comment:
Hi there, I cannot understand why this exception is thrown?

Basically when I try to run the fixer with multiple paths, it complains that I don't have a config. Why would I need a config in that case?

Thanks if you have any answer for that. I could work on a PR if that's a bug, but before I would like to understand.

Reply:
I'm coming from https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4024#issuecomment-750030447 and I didn't want to derail the other issue which is about something else.

> you do not need to change this line, just point which config file to use via cli parameter

Apologies @keradus , I couldn't follow this one.

> it's localization is automatically discovered

I also don't understand this part 😅 "localization" => text translation? 🤔 Nah, doesn't make sense.

---
So maybe this is about ambiguity of resolving possible multiple config files which isn't support, in case multiple files are provided to the fixer?

Because at least the `--help` output would signal to me multiple "paths" are supported:
```
Usage:
  fix [options] [--] [<path>...]
```

I see that there is a dedicated `--config` and usually I see a fallback to `.php_cs.dist`/`.php_cs`. Apologies, I don't quite understand. And the reason I'm asking, I think it would a useful to simple provide multiple files on the command line to fix.

Thank you for your patience with me :)

Reply:
Was answered in https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4024#issuecomment-750288219
> possible since creation of this github issue. just explicitly fingerpoint the config that should be used.

… and in fact like the message says, just use something like this; "ops":
`find … | xargs -n … -P … ./php-cs-fixer.phar --config=…`